explainaboard/constants.py

```diff
@@ -25,7 +25,7 @@ class TaskType(str, Enum):
     cloze_mutiple_choice = "cloze-multiple-choice"
     cloze_generative = "cloze-generative"
     grammatical_error_correction = "grammatical-error-correction"
-    nlg_meta_evaluation = "nlg-meta-evaluation"
+    meta_evaluation_wmt_da = "meta-evaluation-wmt-da"
```

Comment:
Is it safe to change the enum value?

Reply:
What do you mean by the `safe`?
This should be modified since the actual implementation can not reflect such a general task name.
I checked other places to ensure consistent renaming.

Reply:
TaskType is exposed outside, and the change looks to also require changing some values outside the library if the original code is used somewhere. I think it is still okay if TaskType is not encoded into any files.

Reply:
FWIW, I think it's not.

Reply:
>"TaskType is exposed outside, and the change looks to also require changing some values outside the library if the original code is used somewhere. I think it is still okay if TaskType is not encoded into any files."

Thanks!  Since the task `nlg_meta_evaluation` is added relatively recently, so there is little downstream application of it.